spec/views/layout/application.html.erb_spec.rb

```diff
@@ -0,0 +1,140 @@
+    it_behaves_like 'a registered User'
+    before(:each) do
+      @user = FactoryGirl.create(:user)
+      login_as @user
+      visit root_path
+    end
+
+    it 'should link to projects' do
+      expect(page).to have_link('Projects', href: projects_path)
+    end
+
+    it 'should show 4 links' do
+      expect(page).to have_css('nav a', count: 4)
+    end
+  end
+
+  context 'for a superadmin' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      @user = FactoryGirl.create(:user, superadmin: true)
+      login_as @user
+      visit root_path
+    end
+
+    it 'should show 3 links' do
+      expect(page).to have_css('nav a', count: 3)
+    end
+  end
+
+  context 'for a wimi' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      chair = FactoryGirl.create(:chair)
+      wimi_user = FactoryGirl.create(:user)
+      @user = FactoryGirl.create(:wimi, user: wimi_user, chair: chair).user
+      login_as @user
+      visit root_path
+    end
+
+    it 'should link to projects' do
+      expect(page).to have_link('Projects', href: projects_path)
+    end
+
+    it 'should show 4 links' do
+      expect(page).to have_css('nav a', count: 4)
+    end
+  end
+
+  context 'for a hiwi' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      @user = FactoryGirl.create(:user)
+      chair = FactoryGirl.create(:chair)
+      project = FactoryGirl.create(:project, chair: chair, status: true)
+      @user.projects << project
+      login_as @user
+      visit root_path
+    end
+
+    it 'should link to projects' do
+      expect(page).to have_link('Projects', href: projects_path)
+    end
+
+    it 'should show 4 links' do
+      expect(page).to have_css('nav a', count: 4)
+    end
+  end
+
+  context 'for a chair representative' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      chair = FactoryGirl.create(:chair)
+      @user = FactoryGirl.create(:user)
+      FactoryGirl.create(:chair_representative, user: @user, chair: chair)
+      login_as @user
+      visit root_path
+    end
+
+    it 'should link to projects' do
+      expect(page).to have_link('Projects', href: projects_path)
+    end
+
+    it 'should link to chairs' do
+      expect(page).to have_link('Research Groups', href: chairs_path)
+    end
+
+    it 'should show 5 links' do
+      expect(page).to have_css('nav a', count: 5)
+    end
+  end
+
+  context 'for an admin' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      chair = FactoryGirl.create(:chair)
+      @user = FactoryGirl.create(:user)
+      FactoryGirl.create(:chair_wimi, user: @user, chair: chair, admin: true)
+      login_as @user
+      visit root_path
+    end
+
+    it 'should link to projects' do
+      expect(page).to have_link('Projects', href: projects_path)
+    end
+
+    it 'should link to chairs' do
+      expect(page).to have_link('Research Groups', href: chairs_path)
+    end
+
+    it 'should show 5 links' do
```

Comment:
Wie in dem anderen PR schon erwaehnt sind die ` it 'should show 5 links'` tests nicht super sinnvoll denke ich. Die anderen Tests sollten ausreichen.